src/quic/streams.h

```diff
@@ -387,7 +391,79 @@ class Stream final : public AsyncWrap,
   void Schedule(Queue* queue);
   void Unschedule();
 };
+}  // namespace quic
+class DataQueueFeeder final : public AsyncWrap {
```

Comment:
This would need to be in the `node::quic` namespace. It's not clear why it's placed in the `node` namespace

Reply:
Yes, I need some advise. The problem is that the DataQueue stuff is not in node::quic namespace.
The DataQueueFeeder for example is in queue.cc and queue.h. As it needed some local objects defined in queue.cc .
So if you have an idea how to distribute the DataQueue cleaner, I will proceed as I also do not like it.